lib/howl/inspect.moon

```diff
@@ -139,7 +139,12 @@ show_popup = (editor, inspections, pos) ->
     keep_alive: true,
   }
 
-display_inspections = (editor) ->
+display_inspections = ->
+  timer.on_idle (config.display_inspections_delay / 1000), display_inspections
```

Comment:
It's a reoccurring idle timer, so I wouldn't call it a loop. I think it's easier to just use the existing idle timer functionality than to attempt managing the interval / resetting / cursor state handling registering manually, and it's not in practice very inefficient either.